Pages/gamepage.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+
+<html>
+<head>
+  <title>Game Time!</title>
+</head>
+
+<body>
+  <header>
+    <div class= "header-inline">
+      <img src= "https://www.deltavcodeschool.com/wp-content/uploads/deltav-logo.jpg"/>
+      <nav class= "primary-nav">
+        <ul>
+          <li><a href= "index.html">Home</a></li>
+          <li><a href= "eduandwork.html">Education and Work Experience</a></li>
+          <li><a href= "funfacts.html">Fun Facts</a></li>
+          <li><a href= "gamepage.html">Guessing Game</a></li>
+        </ul>
+      </nav>
+    </div>
+    <nav class= "social-nav">
+      <ul>
+        <li><a href= "https://www.facebook.com/dustin.yelland">Facebook</a></li>
+        <li><a href= "https://twitter.com/dustinoid2000">Twitter</a></li>
+        <li><a href= "https://github.com/dustinyschild">Github</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <h1>Lets Play a Game!</h1>
+  <p>
+    Here's the rules: First you will be asked 5 yes or no questions, you get 1 guess each.
+    after that there are 2 additional questions, the first of which you will have 4 guesses to get the right answer.
```

Comment:
New sentence should be capitalized!

![Grammar Police](https://arnoldzwicky.files.wordpress.com/2012/06/grammarpolice.jpg?w=450)